src/content/protocols/compound-v3-arbitrum.md

```diff
@@ -0,0 +1,177 @@
+---
+protocol: "Compound-V3"
+website: "https://compound.finance/"
+x: "https://x.com/compoundfinance"
+github: ["https://github.com/compound-finance/compound-protocol"]
+defillama_slug: ["compound-v3"]
+chain: "Arbitrum"
+stage: 0
+reasons: ["Incorrect Docs"]
+risks: ["M", "H", "L", "H", "M"]
+author: ["mmilien_"]
+submission_date: "2025-02-18"
+publish_date: "1970-01-01"
+acknowledge_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Compound III is an EVM compatible protocol that enables supplying of crypto assets as collateral in order to borrow the base asset. Multiple base assets are supported such as USDC.e, USDC, WETH, and USDT. Accounts can also earn interest by supplying the base asset to the protocol. The market logic of each base asset is implemented in respective `Comet`contracts. Those `Comet`s are deployed by the `Comet Factory` using the `Configurator`. The `Configurator` holds the parameters of each market. A new `Comet` contract needs to be deployed every time the parameters of the market change.
```

Comment:
Compound-v3 is a lending protocol that accepts a base asset as liquidity and allows borrowing this base asset with a variety of other assets as collateral. Multiple base assets are supported such as USDC.e, USDC, WETH, and USDT. Each base asset represents an isolated lending market managed by a separate instance of the protocol. Compound governance is able to update various parameters for each of these markets.